src/util/int_parser.cpp

```diff
@@ -157,4 +157,50 @@ Result<int64_t> to_int(const std::string &num, int base)
   }
 }
 
+SizedType get_integer_type(uint64_t n)
```

Comment:
See comment elsewhere. This does not belong in util because of the dependency cycle it implies.